### PR TITLE
fix(1779) - search-spa fixed dropdown onblur closing

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Fixes

- 1779

# Explain the feature/fix

1. Implemented search spa dropdown onblur on filters

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

https://user-images.githubusercontent.com/31166322/148246815-d400f991-8f90-4246-b340-9ebcd4398873.mov

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/search-spa/src/app/search/search.component.ts [hunk 1]
```diff
@@ -110,4 +132,12 @@ export class SearchComponent implements OnInit {
...
     (document as any).querySelector('opc-feedback').toggle();
   };
+
+  onBlur() {
+    console.log('poda');
+  }
+
+  onConsole(event: Event) {
+    console.log(event);
+  }
```
Comment (riginoommen) [inline packages/search-spa/src/app/search/search.component.ts:142]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/search-spa/src/app/search/search.component.ts (hunk 1 above)
Comment (akhilmhdh) [reply packages/search-spa/src/app/search/search.component.ts]: My bad. I did remove this. Not sure how it came back. Changed

Comment (riginoommen) [reply packages/search-spa/src/app/search/search.component.ts]: Resolved

**Review by riginoommen — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]
LGTM

**Pull request merged**